app-libs/stf/src/getter.rs

```diff
@@ -269,27 +294,92 @@ impl ExecuteGetter for TrustedGetterSigned {
 				Some(attestations.encode())
 			},
 			 */
-			TrustedGetter::ceremonies_aggregated_account_data(
-				who,
-				community_id,
-				_ceremony_index,
-			) => {
-				error!("TrustedGetter ceremonies_aggregated_account_data");
+			TrustedGetter::ceremonies_aggregated_account_data(who, community_id, account_id) => {
+				debug!("TrustedGetter ceremonies_aggregated_account_data");
+				//Todo Master or Me?
+				if !is_ceremony_master(who) {
+					debug!("TrustedGetter ceremonies_aggregated_account_data, return: No master");
+					return None
+				}
```

Comment:
This is a good question, does anything speak against only me being able to get my own aggregated account data?

Reply:
Master calls it do get the meetup index when listing all the attestees https://github.com/encointer/community-sidechain/blob/9c51f2ba1526789c99045b2c83bd36c4db081df8/cli/src/ceremonies/commands/list_attestees.rs#L162. So I don't know. Is it possible that both of them can get this data?

Reply:
Ah I see. Ok, but I must also be able to get my own meetup index to know which meetup I am assigned to, no?